src/Bridges/Nette/Bridge.php

```diff
@@ -47,8 +48,8 @@ public static function renderLatteError(?\Throwable $e): ?array
 					. BlueScreen::highlightLine(htmlspecialchars($e->sourceCode, ENT_IGNORE, 'UTF-8'), $e->sourceLine)
 					. '</div></pre>',
 			];
-
-		} elseif ($e && strpos($file = $e->getFile(), '.latte--')) {
+		}
+		if ($e && strpos($file = $e->getFile(), '.latte--')) {
```

Comment:
why?

Reply:
Suggestion only, but it can be converted into if-branch (clearer intention, lower complexity numbers).